Models/MessageModel.cpp

```diff
@@ -158,7 +158,7 @@ QVariant MessageModel::dataInternal(const QModelIndex &index, int role) const {
     case CppType::CPPTYPE_UINT64: return static_cast<unsigned long long>(refl->GetUInt64(*_protobuf, field));
     case CppType::CPPTYPE_DOUBLE: return refl->GetDouble(*_protobuf, field);
     case CppType::CPPTYPE_FLOAT: return refl->GetFloat(*_protobuf, field);
-    case CppType::CPPTYPE_BOOL: return QVariant();
+    case CppType::CPPTYPE_BOOL: return (role == Qt::EditRole) ? refl->GetBool(*_protobuf, field) : QVariant();
```

Comment:
Why do we *ever* just want to lie about the boolean value being missing?

Reply:
The display role is for display purposes only, we are using `Qt::CheckStateRole` for displaying a checkbox instead.